src/components/Posts/PostForm.tsx

```diff
@@ -51,6 +52,7 @@ const PostForm: FC<PostFormProps> = ({ door, hideForm }) => {
         className="block w-full h-20 p-0 outline-none bg-transparent border-b-2 border-gray-700"
         ref={inputRef}
         placeholder={FORM_PLACEHOLDER}
+        onChange={(event) => setCommentData?.(event.currentTarget.value)}
```

Comment:
Er noe skeptisk til denne, uten at jeg faktisk har kjørt koden selv. I fjor skrev vi om til å kun bruke `inputRef` fordi det var performance problemer ved å oppdatere state på hvert tastetrykk. Bør sørge for at dette enten ikke er et problem nå, eller skrive om til å bruke `inputRef`.

Reply:
Ser ikke ut til å være noe særlig forskjell i performance med den onChange eventen i følge lighthouse. Er enig i at det er mer clean med ref, men etter å ha sett litt på det virker det som det blir litt mongo akkurat her siden input dataen skal leses i en søster-komponent. Fant ingen god løsning på det.